src/js/themes/hpe.js

```diff
@@ -458,7 +446,7 @@ export const hpe = deepFreeze({
     size: {
       small: {
         border: {
-          radius: '6px',
+          radius: '2em',
```

Comment:
Just curious, could we set it at `theme.button.border.radius`?

Reply:
I'll see if we can do this